app/uk/gov/hmrc/apipublisher/models/PublisherRequest.scala

```diff
@@ -20,6 +20,35 @@ import play.api.libs.json._
 import uk.gov.hmrc.apipublisher.models.APICategory.{APICategory, formatAPICategory}
 import uk.gov.hmrc.http.UnprocessableEntityException
 
+sealed trait ApiVersionSource {
+  def asText: String
+}
+
+object ApiVersionSource {
+  case object RAML extends ApiVersionSource {
+    val asText = "RAML"
+  }
+  case object OAS extends ApiVersionSource {
+    val asText = "OAS"
+  }
+
+  case object UNKNOWN extends ApiVersionSource {
+    val asText = "UNKNOWN"
+  }
+
+  implicit val format: Format[ApiVersionSource] = new Format[ApiVersionSource] {
+    def reads(json: JsValue): JsResult[ApiVersionSource] = json match {
+      case JsString(RAML.asText) => JsSuccess(RAML)
+      case JsString(OAS.asText) => JsSuccess(OAS)
+      case e => JsError(s"Cannot parse source value from '$e'")
```

Comment:
Presumably there's no formatter for Unknown deliberately?

Reply:
No - this one got missed.  Probably pre-dates the UNKNOWN